idd/Energy+.idd.in

```diff
@@ -23564,6 +23572,70 @@ AirflowNetwork:Distribution:Linkage,
       \note The zone name is where AirflowNetwork:Distribution:Component:Duct is exposed. Leave this field blank if the duct
       \note conduction loss is ignored.
 
+AirflowNetwork:OccupantVentilationControl,
+      \memo This object is used to provide advanced thermal comfort control of window opening and closing 
+      \memo for both exterior and interior windows.
+  A1, \field Name
+      \required-field
+      \type object-list
+      \object-list AirflowNetwork:OccupantVentilationControlNames
+      \note Enter the name where the advanced thermal comfort control is required.
```

Comment:
@lgu1234 @jasondegraw The above fourlines should be replaced by 
\required-field
\type alpha
\reference AirflowNetworkOccupantVentilationControlNames